test/api/liskApi.js

```diff
@@ -218,460 +222,177 @@ describe('Lisk.api()', () => {
-			(callback.calledWith(expectedRessult)).should.be.true();
+				LSK.getAccount(address, callback);
+				(LSK.sendRequest.calledWithExactly(GET, 'accounts', options, callback)).should.be.true();
+			});
 		});
-	});
 
-	describe('#listMultisignatureTransactions', () => {
-		it('should list all current not signed multisignature transactions', () => {
-			return liskApi().listMultisignatureTransactions((result) => {
-				(result).should.be.ok();
-				(result).should.be.type('object');
+		describe('#listMultisignatureTransactions', () => {
+			it('should list all current not signed multisignature transactions', () => {
+				return liskApi().listMultisignatureTransactions((result) => {
+					(result).should.be.ok();
+					(result).should.be.type('object');
+				});
 			});
 		});
-	});
 
-	describe('#getMultisignatureTransaction', () => {
-		it('should get a multisignature transaction by id', () => {
-			return liskApi().getMultisignatureTransaction('123', (result) => {
-				(result).should.be.ok();
-				(result).should.be.type('object');
+		describe('#getMultisignatureTransaction', () => {
+			it('should get a multisignature transaction by id', () => {
+				return liskApi().getMultisignatureTransaction('123', (result) => {
+					(result).should.be.ok();
+					(result).should.be.type('object');
+					(result).should.have.property('error').be.eql('Transaction not found');
+				});
 			});
 		});
-	});
 
-	describe('#sendLSK', () => {
-		const expectedResponse = {
-			body: { success: true, transactionId: '8921031602435581844' },
-		};
-		it('should send testnet LSK', () => {
-			const options = {
-				ssl: false,
-				node: '',
-				randomPeer: true,
-				testnet: true,
-				port: testPort,
-				bannedPeers: [],
-			};
-			const callback = sinon.spy();
-			const LSKnode = liskApi(options);
-			const secret = 'soap arm custom rhythm october dove chunk force own dial two odor';
-			const secondSecret = 'spider must salmon someone toe chase aware denial same chief else human';
-			const recipient = '10279923186189318946L';
-			const amount = 100000000;
-			sinon.stub(LSKnode, 'sendRequest').callsArgWith(3, expectedResponse);
-
-			LSKnode.sendLSK(recipient, amount, secret, secondSecret, callback);
-
-			(LSKnode.sendRequest.calledWith(POST, 'transactions', {
-				recipientId: recipient,
-				amount,
-				secret,
-				secondSecret,
-			})).should.be.true();
+		describe('#sendLSK', () => {
+			it('should send testnet LSK', () => {
+				const recipientId = '10279923186189318946L';
 
-			(callback.called).should.be.true();
-			(callback.calledWith(expectedResponse)).should.be.true();
-			LSKnode.sendRequest.restore();
+				LSK.sendLSK(recipientId, defaultAmount, defaultSecret, defaultSecondSecret, callback);
+				(LSK.sendRequest.calledWithExactly(POST, 'transactions', {
+					recipientId,
+					amount: defaultAmount,
+					secret: defaultSecret,
+					secondSecret: defaultSecondSecret,
+				}, callback)).should.be.true();
```

Comment:
This object should be put in an `options` variable to conform to the pattern of the other tests.